packages/bot-cleaning/src/strategies.ts

```diff
@@ -0,0 +1,47 @@
+import { Market } from "@mangrovedao/mangrove.js";
+import Big from "big.js";
+
+type SnipeParams = {
+  takerWants: Big;
+  takerGives: Big;
+};
+
+/**
+ * This function calculates the minimum assets that a taker is willing to exchange, considering whether it's a bid or ask offer.
+ * It starts by setting `takerWants` to the smallest amount of `outboundTkn` that an offer can send,
+ * which is determined by `1 / (10**(outboundTkn.decimals))`. Then, it computes `takerGives` by multiplying
+ * this value by the price (wants/gives). It checks whether the computed amount is greater than the minimum possible volume of `inboundTkn`,
+ * which is `1 / (10**(inboundTkn.decimals))`. If it's not, the logic is reversed: it first tries to
+ * set `takerGives` to the minimum volume of `inboundTkn`, and then it calculates `takerWants` by multiplying this value by the inverse of the price.
```

Comment:
Not sure this ChatGPT'ification of the code is helpful ;-)

Reply:
It's ChatGPTed description I wrote